[4.0.6] Обновление до новой версии

Kostelano opened the pull request:
На 18.01 запланирован выпуск Joomla 4.0.6.

https://github.com/joomla/joomla-cms/pulls?q=is%3Apr+milestone%3A%22Joomla+4.0.6%22+is%3Aclosed+label%3A%22Language+Change%22

Kostelano committed:
Версия в XML

Kostelano committed:
Новое (1)

https://github.com/joomla/joomla-cms/pull/36350 +
https://github.com/joomla/joomla-cms/pull/36267 +
https://github.com/joomla/joomla-cms/pull/36471 +
https://github.com/joomla/joomla-cms/pull/36470 +
https://github.com/joomla/joomla-cms/pull/36487 +
https://github.com/joomla/joomla-cms/pull/36492 +
https://github.com/joomla/joomla-cms/pull/36312 + (из Joomla 3)

b2z commented:
Это будем править?
https://github.com/joomla/joomla-cms/pull/36487/files

Kostelano commented:
@b2z Первый PR - упустил, спасибо, поправлю. Обычно перепроверяю себя перед выпуском релиза.
Второй - не думаю, что нужно править. Порой логика изменения мне неясна. Браян убирает заголовок **Расширения** здесь, но оставляет этот же заголовок в соседних строчках к этому же разделу. Получается нелогичность и разные заголовки.

```
COM_INSTALLER_HEADER_MANAGE="Расширения: Управление"
COM_INSTALLER_HEADER_UPDATE="Расширения: Обновление"
```

b2z commented:
ОК. Тут ещё грохнули неиспользуемые
https://github.com/joomla/joomla-cms/pull/36470/files

Kostelano commented:
Я лажанул. На самом деле я и 1-й ПР вносил изменения на ПК, но потерял файл при затягивании. Там же были изменения из последнего ПР. Поправлю дома.

Kostelano committed:
Новое (2)

https://github.com/joomla/joomla-cms/pull/36470 +
https://github.com/joomla/joomla-cms/pull/36487 +

Kostelano committed:
Фикс из-за copy/paste

Kostelano commented on administrator/language/ru-RU/com_installer.ini:
```diff
@@ -1,303 +1,303 @@
-COM_INSTALLER_MSG_WARNINGS_SMALLPOSTSIZEDESC="Максимальный размер POST-запроса определяет общее количество информации, которое можно передать на сервер. В настоящий момент установлено значение менее 16MB, что может создать проблемы при сохранении больших материалов или загрузке расширений. Пожалуйста, измените значение переменной <strong>post_max_size</strong> в файле <strong>php.ini</strong>."
-COM_INSTALLER_MSG_WARNINGS_SMALLUPLOADSIZE="Ограничен максимальный размер загружаемого файла"
-COM_INSTALLER_MSG_WARNINGS_SMALLUPLOADSIZEDESC="Максимальный размер загружаемого файла составляет менее 16MB. При попытке установки расширений могут возникнуть проблемы. Пожалуйста, измените значения переменных <strong>post_max_size</strong> и <strong>upload_max_filesize</strong> в файле <strong>php.ini</strong> (и/или <strong>.htaccess</strong>). Обратите внимание, что значение <strong>post_max_ ...
-COM_INSTALLER_MSG_WARNINGS_UPDATE_NOTICE="Перед обновлением убедитесь в совместимости установленных расширений с новой версией Joomla. Настоятельно рекомендуется создать резервную копию базы данных и файлов сайта перед началом обновления."
```
Исправил только эту строку ^^
